### PR TITLE
Add Translator

### PR DESCRIPTION
**Pull request by NetheriteMiner**
There is some slight buggy behavior when translating from English but only when translating multiple sentences. I will fix this eventually, but it should not be a big problem in the meantime.

### CODE REVIEW

**Comment by NetheriteMiner** [comment]
Fixed translating from Other -> English always failing.
Fixed the extra characters when translating multiple sentences in English -> Other.

**Review by hannibal002 — CHANGES_REQUESTED** [review]
The current handling of click event will break other click events on player messages from other mods, e.g., neu's open PV feature.
How about we only replace the message part with the translator click and keep the PV open click action on the player name part?

**Comment by hannibal002** [comment]
image
This bug still happens with neu alpha 19. 
Tested outside dev environment

File: src/main/java/at/hannibal2/skyhanni/features/chat/Translator.kt [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+            if (!SkyHanniMod.feature.chat.translator) return
+            var message = ""
+            for (i in args) {
+                message = "$message$i "
+            }
+
+            val translation = getTranslationToEnglish(message)
+
+            if (translation == "Unable to translate!") LorenzUtils.chat("§c[SkyHanni] Unable to translate message :(")
+            else LorenzUtils.chat("§6[SkyHanni] §eFound translation: $translation")
```
Comment (jani270) [inline src/main/java/at/hannibal2/skyhanni/features/chat/Translator.kt]: pls fix color code

**Review by jani270 — CHANGES_REQUESTED** [review]

File: src/main/java/at/hannibal2/skyhanni/features/chat/Translator.kt (hunk 1 above)
Comment (Rohan-Sangani) [reply src/main/java/at/hannibal2/skyhanni/features/chat/Translator.kt]: which color code needs changing? the gold §6?

Comment (jani270) [reply src/main/java/at/hannibal2/skyhanni/features/chat/Translator.kt]: correct

**Review by hannibal002 — APPROVED** [review]
causes neu problems in dev env, but production is working fine so it doesnt matter

**Pull request merged**